documentation/md/events.md

```diff
@@ -102,6 +102,9 @@ These events are custom to Cytoscape.js, and they occur on the core.
  * `destroy` : when the instance of Cytoscape.js was explicitly destroyed by calling `.destroy()`.
  * `render` : when the viewport is (re)rendered
  * `pan` : when the viewport is panned
+ * `dragpan` : when the viewport is panned via dragging
  * `zoom` : when the viewport is zoomed
+ * `pinchzoom` : when the viewport is zoomed via pinch gesture. This event is only supported for browsers which support the [gesturechange event](https://developer.mozilla.org/en-US/docs/Web/API/Element/gesturechange_event). Other browsers will fire the `scrollzoom` event.
```

Comment:
Or touch events: https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent

Reply:
It's been updated, let me know if it needs further changes.